tests/003-test-override-signature-wipes-existing-signatures.sh

```diff
@@ -0,0 +1,65 @@
+cleanup() {
+  local vg_name=$1
+  local devs=$2
+
+  vgremove -y $vg_name >> $LOGS 2>&1
+  remove_pvs "$devs"
+  remove_partitions "$devs"
+  clean_config_files
+  wipe_signatures "$devs"
+}
```

Comment:
At this point should consider a `libtest.sh` like we have in ostree: https://github.com/ostreedev/ostree/blob/master/tests/libtest.sh

The individual tests source it for funcitons.